test/stub.go

```diff
@@ -98,6 +98,16 @@ func (s *StubBoletoRequest) WithBuyerZipCode(zipcode string) *StubBoletoRequest
 	return s
 }
 
+func (s *StubBoletoRequest) WithRecipientDocumentType(DocumentType string) *StubBoletoRequest {
```

Comment:
Opa, o padrão de parâmetro da função é Camel case 😀  
```suggestion
func (s *StubBoletoRequest) WithRecipientDocumentType(documentType string) *StubBoletoRequest {
```